cluster/proto/cluster/message.proto

```diff
@@ -42,6 +42,7 @@ service ClusterService {
   rpc NotifyPeer(NotifyPeerRequest) returns (NotifyPeerResponse) {}
   rpc Apply(ApplyRequest) returns (ApplyResponse) {}
 
+  rpc GetReadOnlyClass(GetReadOnlyClassRequest) returns (GetReadOnlyClassResponse) {}
```

Comment:
I am tempted to add a different `Apply` endpoint. For example `SchemaQuery` where we'd have a similar demux with an enum. This would help to extend it in the future to proxy more schema queries if we want to.

Reply:
This sounds good to me. Consider naming the request `Query` instead of `SchemaQuery` because it is understood from the context.